docs/configuration.rst

```diff
@@ -7,380 +7,343 @@
-       Example for UDS: ``DD_TRACE_AGENT_URL=unix:///var/run/datadog/apm.socket``
-
-       .. _dd-dogstatsd-url:
-   * - ``DD_DOGSTATSD_URL``
-     - URL
-     - ``unix:///var/run/datadog/dsd.socket`` if available 
-       otherwise ``udp://localhost:8125``
-     - The URL to use to connect the Datadog agent for Dogstatsd metrics. The url can start with
-       ``udp://`` to connect using UDP or with ``unix://`` to use a Unix
-       Domain Socket.   
-       Example for UDP url: ``DD_TRACE_AGENT_URL=udp://localhost:8125``
-       Example for UDS: ``DD_TRACE_AGENT_URL=unix:///var/run/datadog/dsd.socket``
-
-       .. _dd-trace-agent-timeout-seconds:
-   * - ``DD_TRACE_AGENT_TIMEOUT_SECONDS``
-     - Float
-     - 2.0
-     - The timeout in float to use to connect to the Datadog agent.
-
-       .. _dd-trace-writer-buffer-size-bytes:
-   * - ``DD_TRACE_WRITER_BUFFER_SIZE_BYTES``
-     - Int
-     - 8388608
-     - The max size in bytes of traces to buffer between flushes to the agent.
-
-       .. _dd-trace-writer-max-payload-size-bytes:
-   * - ``DD_TRACE_WRITER_MAX_PAYLOAD_SIZE_BYTES``
-     - Int
-     - 8388608
-     - The max size in bytes of each payload item sent to the trace agent. If the max payload size is greater than buffer size, then max size of each payload item will be the buffer size.
-
-       .. _dd-trace-writer-interval-seconds:
-   * - ``DD_TRACE_WRITER_INTERVAL_SECONDS``
-     - Float
-     - 1.0
-     - The time between each flush of traces to the trace agent.
-
-       .. _dd-trace-startup-logs:
-   * - ``DD_TRACE_STARTUP_LOGS``
-     - Boolean
-     - False
-     - Enable or disable start up diagnostic logging.
-
-       .. _dd-trace-sample-rate:
-   * - ``DD_TRACE_SAMPLE_RATE``
-     - Float
-     - 1.0
-     - A float, f, 0.0 <= f <= 1.0. f*100% of traces will be sampled.
+
+.. ddtrace-configuration-options::
+   DD_ENV:
+     description: |
+         Set an application's environment e.g. ``prod``, ``pre-prod``, ``staging``. Added in ``v0.36.0``. See `Unified Service Tagging`_ for more information.
+
+   DD_SERVICE:
+     default: (autodetected)
+     description: |
+         Set the service name to be used for this application. A default is
+         provided for these integrations: :ref:`bottle`, :ref:`flask`, :ref:`grpc`,
+         :ref:`pyramid`, :ref:`pylons`, :ref:`tornado`, :ref:`celery`, :ref:`django` and
+         :ref:`falcon`. Added in ``v0.36.0``. See `Unified Service Tagging`_ for more information.
+
+   DD_SERVICE_MAPPING:
+     description: |
+         Define service name mappings to allow renaming services in traces, e.g. ``postgres:postgresql,defaultdb:postgresql``.
+
+   DD_TAGS:
+     description: |
+         Set global tags to be attached to every span. Value must be either comma or space separated. e.g. ``key1:value1,key2,value2`` or ``key1:value key2:value2``.
+         Comma separated support added in ``v0.38.0`` and space separated support added in ``v0.48.0``.
```

Comment:
could we add `version_added:` as a key in these? that would be awesome for helping remind us to add that valuable field.

Reply:
ooooo! yes, that is a great idea!

Reply:
```yaml
version_added:
  0.38.0: comma separated support added
  0.48.0: space separated support added
```

Then render at the bottom as:

```md
_Added in version v0.38.0_: comma separated support added

_Added in version v0.48.0_: space separated support added
```

?